test/support/TestConnector.scala

```diff
@@ -40,6 +40,9 @@ class TestConnector @Inject() (httpClient: HttpClient)(implicit executionContext
   def find(id: TpsId)(implicit hc: HeaderCarrier): Future[TpsPayments] =
     httpClient.GET[TpsPayments](s"http://localhost:$port/tps-payments-backend/find/id/${id.value}", headers)
 
+  def paymentItem(id: PaymentItemId)(implicit hc: HeaderCarrier): Future[TpsPaymentItem] =
```

Comment:
Can we please rename it to `getPaymentItem` to express more what it actually does. A lack of verb in this methods makes me hard to understand the code.

Reply:
Good shout!